worker/src/main/java/gov/cms/ab2d/worker/properties/PropertiesInit.java

```diff
@@ -12,9 +12,18 @@
 @Slf4j
 public class PropertiesInit {
 
-    // Load all of the properties from the database
+    private ConfigurableEnvironment configurableEnvironment;
+
+    private PropertiesService propertiesService;
+
     public PropertiesInit(PropertiesService propertiesService, ConfigurableEnvironment configurableEnvironment) {
-        log.info("Adding properties");
+        this.propertiesService = propertiesService;
+        this.configurableEnvironment = configurableEnvironment;
+    }
+
+    // Load all of the properties from the database and insert/overwrite
+    public void updatePropertiesFromDatabase() {
+        log.info("Updating properties");
         final Map<String, Object> properties = propertiesService.getAllProperties().stream()
                 .collect(Collectors.toMap(Properties::getKey, Properties::getValue));
         configurableEnvironment.getPropertySources().addLast(new MapPropertySource("application", properties));
```

Comment:
The problem is that even though the properties are reloaded and added to the `configurableEnvironment`, the beans have already been wired. Property changes won't be reflected in beans and thus will have no effect. 

Take a look at the last 3 paragraphs in https://jira.cms.gov/browse/AB2D-1035. Each property change might need a small amount of handling logic.